command/operator_rekey.go

```diff
@@ -708,33 +721,56 @@ func (c *OperatorRekeyCommand) printUnsealKeys(client *api.Client, status *api.R
 	switch status.VerificationRequired {
 	case false:
 		c.UI.Output("")
-		c.UI.Output(wrapAtLength(fmt.Sprintf(
-			"Vault rekeyed with %d key shares and a key threshold of %d. Please "+
-				"securely distribute the key shares printed above. When Vault is "+
-				"re-sealed, restarted, or stopped, you must supply at least %d of "+
-				"these keys to unseal it before it can start servicing requests.",
-			status.N,
-			status.T,
-			status.T)))
+		switch strings.ToLower(strings.TrimSpace(c.flagTarget)) {
+		case "barrier":
+			c.UI.Output(wrapAtLength(fmt.Sprintf(
+				"Vault rekeyed with %d key shares and a key threshold of %d. Please "+
+					"securely distribute the key shares printed above. When Vault is "+
+					"re-sealed, restarted, or stopped, you must supply at least %d of "+
+					"these keys to unseal it before it can start servicing requests.",
+				status.N,
+				status.T,
+				status.T)))
+		case "recovery", "hsm":
+			c.UI.Output(wrapAtLength(fmt.Sprintf(
+				"Vault rekeyed with %d key shares and a key threshold of %d. Please "+
```

Comment:
Can we get the target into the text here and above, e.g. "Vault %s keys rekeyed with %d..." where %s is "unseal" or "recovery"?

Reply:
added